Missing Type in ShowMessageRequestParams

### Describe the bug

When the client sends a `new-scala-file` command Metals omits to send the `type` field in the response, as you can see in LSP logs bellow: 
```
2023-01-11T00:29:38.888 helix_lsp::transport [INFO] -> {"jsonrpc":"2.0","method":"workspace/executeCommand","params":{"arguments":[],"command":"new-scala-file"},"id":12}
2023-01-11T00:29:38.912 helix_lsp::transport [INFO] <- {"jsonrpc":"2.0","id":"3","method":"window/showMessageRequest","params":{"actions":[{"title":"scala-file"},{"title":"scala-class"},{"title":"scala-case-class"},{"title":"scala-object"},{"title":"scala-trait"},{"title":"scala-package-object"},{"title":"scala-worksheet"},{"title":"scala-script"}],"message":"Select the kind of file to create"}}
2023-01-11T00:29:38.912 helix_term::application [ERROR] received malformed method call from Language Server: window/showMessageRequest: protocol error: InvalidParams: Invalid params: missing field `type`.
```
This is crashing Helix because it doesn't default to anything.
I guess the LSP Java API allows to create a ShowMessageRequestParams without having to call `setType`.


### Expected behavior

`type` field should be always there as per the spec https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#showMessageRequestParams

### Operating system

macOS

### Editor/Extension

Helix

### Version of Metals

v.11.10

### Extra context or search terms

LSP

--- a/metals/src/main/scala/scala/meta/internal/metals/clients/language/ConfiguredLanguageClient.scala
+++ b/metals/src/main/scala/scala/meta/internal/metals/clients/language/ConfiguredLanguageClient.scala
@@ -143,6 +143,7 @@ final class ConfiguredLanguageClient(
     val result = new ShowMessageRequestParams()
     result.setMessage(params.placeHolder)
     result.setActions(params.items.map(item => new MessageActionItem(item.id)))
+    result.setType(MessageType.Info)
     result
   }
 

